From 26434fb4376a5db1f38fca8807825fbabc81be7b Mon Sep 17 00:00:00 2001
From: Andrew Deason <adeason@sinenomine.net>
Date: Sat, 4 Apr 2020 22:28:21 -0500
Subject: [PATCH 1/5] Add autoconf-archive to src/external

Add autoconf-archive to the src/external mechanism, so we can more
easily import and update the AX_* m4 macros we pull in from
autoconf-archive. Commits are imported from
<git://git.savannah.gnu.org/autoconf-archive.git>.

We already have a copy of ax_gcc_func_attribute.m4 in the tree, so
include that in the list of files. While we're here, also include a
few more macros for checking compiler flags, which will be used in
subsequent commits.

Change-Id: I8c6288fc1d48a47837ca08f8b9207e0ada921af8
Reviewed-on: https://gerrit.openafs.org/14133
Reviewed-by: Cheyenne Wills <cwills@sinenomine.net>
Tested-by: BuildBot <buildbot@rampaginggeek.com>
Reviewed-by: Benjamin Kaduk <kaduk@mit.edu>
(cherry picked from commit a072c65bba86cbcd81157e354d3719ac41a2c97d)
---
 src/external/autoconf-archive-author | 1 +
 src/external/autoconf-archive-files  | 5 +++++
 2 files changed, 6 insertions(+)
 create mode 100644 src/external/autoconf-archive-author
 create mode 100644 src/external/autoconf-archive-files

diff --git a/src/external/autoconf-archive-author b/src/external/autoconf-archive-author
new file mode 100644
index 000000000..1c668c50b
--- /dev/null
+++ b/src/external/autoconf-archive-author
@@ -0,0 +1 @@
+Autoconf Archive Maintainers <autoconf-archive-maintainers@gnu.org>
diff --git a/src/external/autoconf-archive-files b/src/external/autoconf-archive-files
new file mode 100644
index 000000000..42c7c3852
--- /dev/null
+++ b/src/external/autoconf-archive-files
@@ -0,0 +1,5 @@
+m4/ax_append_compile_flags.m4	m4/ax_append_compile_flags.m4
+m4/ax_append_flag.m4		m4/ax_append_flag.m4
+m4/ax_check_compile_flag.m4	m4/ax_check_compile_flag.m4
+m4/ax_gcc_func_attribute.m4	m4/ax_gcc_func_attribute.m4
+m4/ax_require_defined.m4	m4/ax_require_defined.m4
-- 
2.35.1

